source/state_representation/include/state_representation/trajectory/TrajectoryBase.hpp

```diff
@@ -388,7 +356,7 @@ inline const std::chrono::nanoseconds TrajectoryBase<TrajectoryT>::get_trajector
 
 template<typename TrajectoryT>
 inline unsigned int TrajectoryBase<TrajectoryT>::get_size() const {
-  this->assert_trajectory_not_empty();
+  this->assert_not_empty(); // TBD, we also don't throw for get_reference_frame/get_joint_names on empty state
```

Comment:
I'm on the side of not throwing here

Reply:
yeah, that would be ok I think. The tests will need a small adjustment but let's go for it